docs/Expensify-Card-revenue share for ExpensifyApproved!-partners.md

```diff
@@ -9,9 +9,14 @@ Start making more with us! We're thrilled to announce a new incentive for our US
 <br>This program is currently only available to US-based ExpensifyApproved! partner accountants.
 
 # How-to
-To benefit from this program, all you need to do is ensure that you are listed as a domain admin on your client's Expensify account. If you're not currently a domain admin, your client can follow the instructions outlined in [our help article](https://community.expensify.com/discussion/5749/how-to-add-and-remove-domain-admins#:~:text=Domain%20Admins%20have%20total%20control,a%20member%20of%20the%20domain.) to assign you this role.
+To benefit from this program, complete the following steps 
+1. Ensure that you are listed as the Primary Contact under your client's domain in Expensify. If you're not currently the Primary Contact for your client, you or your client can follow the instructions outlined in [our help article](https://community.expensify.com/discussion/5749/how-to-add-and-remove-domain-admins#:~:text=Domain%20Admins%20have%20total%20control,a%20member%20of%20the%20domain.) to assign you this role.
+2. Add a Business Bank Account to the Primary Contact account. Follow the instructions in [our help article](https://community.expensify.com/discussion/4751/how-to-add-a-verified-bank-account-us-only) to get a Business Bank Account added. 
```

Comment:
It doesn’t have to be a verified bank account. Just business. So I think the link is wrong?